src/styles/blocks/content.scss

```diff
@@ -0,0 +1,16 @@
+.content {
+  display: grid;
+  justify-content: center;
+
+  &__container {
+    width: 944px;
+    height: 862px;
+    padding: 50px 40px 72px;
+
+    display: grid;
+    grid-template-columns: repeat(4, 198px);
```

Comment:
What is 198px? Create a variable for this value, also I guess it should be 200, not 198